services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchRulesSpec.groovy

```diff
@@ -421,6 +427,61 @@ class SwitchRulesSpec extends BaseSpecification {
         flowHelper.deleteFlow(flow.id)
     }
 
+    def "Traffic counters for ingress rules are reset on flow rerouting"() {
+        given: "Two active switches and two possible flow paths at least"
+        def switches = topology.getActiveSwitches()
```

Comment:
I think we really need to have some kind of library which would handle these flow route selection situations for us. Because this same code is used in a lot of test cases and if something changes in how the Kilda works - it'd be a lot to fix.
Something like def findPaths(int minDistance = 1, int minAlternatePaths, int maxAlternatePaths) would be great.